config/dev/spm/sku/sival/BUILD.bazel

```diff
@@ -3,21 +3,276 @@
 # SPDX-License-Identifier: Apache-2.0
 
 load("@rules_pkg//pkg:tar.bzl", "pkg_tar")
+load(
+    "//rules:hsm.bzl",
+    "hsm_certgen",
+    "hsm_config_script",
+    "hsm_key_template",
+)
+load(
+    "//rules:hsmtool.bzl",
+    "hsmtool_aes_keygen",
+    "hsmtool_ecdsa_keygen",
+    "hsmtool_generic_keygen",
+    "hsmtool_pk11_attrs",
+    "hsmtool_rsa_keygen",
+)
+
+package(default_visibility = ["//visibility:public"])
+
+hsm_key_template(
+    name = "sival-aes-wrap-v0",
+    import_template = hsmtool_pk11_attrs({
+        "CKA_DECRYPT": True,
+        "CKA_ENCRYPT": False,
+        "CKA_SENSITIVE": True,
+        "CKA_TOKEN": True,
+        "CKA_UNWRAP": True,
+        "CKA_WRAP": False,
+        "CKA_EXTRACTABLE": False,
+    }),
+    keygen_params = hsmtool_aes_keygen(
+        label = "sival-aes-wrap-v0",
+        template = {
+            "CKA_ENCRYPT": True,
+            "CKA_DECRYPT": True,
+            "CKA_WRAP": True,
+            "CKA_UNWRAP": True,
+            "CKA_SENSITIVE": True,
+            "CKA_EXTRACTABLE": True,
+            "CKA_TOKEN": True,
+        },
+    ),
+    type = "aes",
+    wrapping_key = "//config/dev/spm/sku:spm-rsa-wrap-v0",
+    wrapping_mechanism = "RsaPkcs",
+)
+
+hsm_key_template(
+    name = "sku-sival-rsa-rma-v0",
+    export_public_only = True,
+    import_template_public = hsmtool_pk11_attrs({
+        "CKA_ENCRYPT": True,
+        "CKA_TOKEN": True,
+        "CKA_WRAP": True,
+    }),
+    keygen_params = hsmtool_rsa_keygen(
+        extractable = False,
+        key_length = 3072,
+        label = "sku-sival-rsa-rma-v0",
+        private_template = {
+            "CKA_CLASS": "CKO_PRIVATE_KEY",
+            "CKA_LABEL": "sku-sival-rsa-rma-v0.priv",
+            "CKA_DECRYPT": True,
+            "CKA_SIGN": True,
+            "CKA_TOKEN": True,
+            "CKA_EXTRACTABLE": True,
+        },
+        public_exponent = 65537,
+        public_template = {
+            "CKA_CLASS": "CKO_PUBLIC_KEY",
+            "CKA_LABEL": "sku-sival-rsa-rma-v0.pub",
+            "CKA_ENCRYPT": True,
+            "CKA_VERIFY": True,
+            "CKA_TOKEN": True,
+        },
+        wrapping = True,
+    ),
+    type = "rsa",
+)
+
+hsm_key_template(
+    name = "opentitan-ca-root-v0",
+    export_public_only = True,
+    keygen_params = hsmtool_ecdsa_keygen(
+        curve = "1.2.840.10045.3.1.7",
```

Comment:
nit: it would be nice if these were mapped to human readable bazel constants, like "ECC_CURVE_P256" or something like that

Reply:
Hehe realized @cfrantz and I said the same thing

Reply:
Fixed.